Added Wither, Carrots Block, Potatoes Block, and Cobble/Mossy walls.

Sleaker opened the pull request:
Signed-off-by: Sleaker sleaker@gmail.com

Darsstar commented on src/main/java/org/spout/vanilla/component/living/hostile/Wither.java:
```diff
@@ -0,0 +1,43 @@
+import org.spout.vanilla.protocol.entity.living.WitherEntityProtocol;
+
+/**
+ * A component that identifies the entity as an EnderDragon.
```
Will a Wither launch its 'missiles' when it thinks it is an EnderDragon? :p

Sleaker commented:
@Darsstar crazy pills man.. you're taking crazy pills

Sleaker committed:
Added Wither, Carrots Block, Potatoes Block, and Cobble/Mossy walls.
Added FlowerPotBlock, FlowerPot, made proper materials attach to
FlowerPot.
Fixed wheat drop values being incorrect.
Removed unused imports.

Signed-off-by: Sleaker <sleaker@gmail.com>
